tests/helper/helper.go

```diff
@@ -139,6 +140,22 @@ func WaitForDeploymentReplicaCount(t *testing.T, kc *kubernetes.Clientset, name,
 	return false
 }
 
+// Waits for number of iterations and returns replica count.
+func WaitForDeploymentReplicaCountChange(t *testing.T, kc *kubernetes.Clientset, name, namespace string, iterations, intervalSeconds int) int {
+	t.Log("Waiting for some time to see if deployment replica count changes")
+	for i := 0; i < iterations; i++ {
+		deployment, _ := kc.AppsV1().Deployments(namespace).Get(context.Background(), name, metav1.GetOptions{})
+		replicas := deployment.Status.Replicas
+
+		t.Logf("Deployment - %s, Current  - %d", name, replicas)
+
+		time.Sleep(time.Duration(intervalSeconds) * time.Second)
+	}
```

Comment:
Shouldn't we break the loop if the replicas have changed? I mean, if `replicas` changes in the first loop, we don't need to continue, right?

Reply:
Yeah, my bad.